scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -602,6 +602,12 @@ def add_parse_arguments(parser = None):
         help="""Create a report with a custom name.
         """)
 
+    parser.add_argument(
+        "--report-summary", action="store", nargs='?', type=int, const=10,
```

Comment:
hmm, how it works with `--report-summary 0`, or what if someone wants to get report for all the issues ?